### DISCUSSION TITLE
Update custom user column with button

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8280] [Help] [answered]
Hey,

is it possible to update a user column with a button in the app?

**Comment by melohagan** [comment, marked as answer]
Hey @kasgithub8 

No it's not. The user's table is a special read-only table (https://docs.budibase.com/docs/users-table) that can only be updated via **User Management**

https://docs.budibase.com/docs/user-management

**Reply by kasgithub8** [reply]
Thanks @melohagan.

Unlucky tho.
